pkgs/by-name/ni/nixos-build-vms/build-vms.nix

```diff
@@ -9,9 +9,9 @@ let
     imports = [ module ];
   }) (import networkExpr);
 
-  pkgs = import ../../../../.. { inherit system config; };
+  pkgs = import ../../../.. { inherit system config; };
 
-  testing = import ../../../../lib/testing-python.nix {
+  testing = import ../../../../nixos/lib/testing-python.nix {
```

Comment:
I'm not behind my laptop now, but you might be able to get this as a callPackage-style argument?

Reply:
Honestly it should be a channel reference because nixos-build-vms already refers to _this very file_ through one.

Reply:
Done.

Reply:
Added a better (worse?) hack that satisfies nixpkgs-vet.